docs/markdown/test.md

```diff
@@ -0,0 +1,116 @@
+---
+title: How to work with us
```

Comment:
Added Front Matter support. We can expand this to add more properties via markdown